zingo-testutils/src/lightclient/with_assertions.rs

```diff
@@ -69,10 +71,11 @@ where
     environment.bump_chain().await;
 
     recipient.do_sync(false).await.unwrap();
-    assert_send_outputs_match_recipient(recipient, &proposal, &txids).await;
+    get_proposal_vs_records_matched_total_output_value(recipient, &proposal, &txids).await;
```

Comment:
actually, im afraid somebody will see this line does nothing, not realize it asserts, and delete it. it needs to start with 'assert